teaser/examples/e9_building_data_import_from_excel.py

```diff
@@ -0,0 +1,629 @@
+def import_building_from_excel(
+    project, building_name, construction_age, path_to_excel, sheet_names
+):
+    """
+    Import building data from excel, convert it via the respective zoning and feed it to teasers logic classes.
+    Pay attention to hard coded parts, which are marked.
+
+    :param project: Teaser project object
+    :type project:
+    :param building_name: name of building to be set in the project
+    :type building_name:
+    :param construction_age: construction age of the building
+    :type construction_age: int
+    :param path_to_excel: path to excel file to be imported
+    :type path_to_excel: str
+    :param sheet_names: sheet names which shall be imported
+    :type sheet_names: str or list
+    :return: Teaser project
+    :rtype:
+    """
+
+    def warn_constructiontype(element):
+        if element.construction_type is None:
+            warnings.warn(
+                'In Zone "%s" the %s construction "%s" could not be loaded from the TypeBuildingElements.json, '
+                "an error will occur due to missing data for calculation."
+                "Check for spelling and the correct combination of building age and construction type."
+                "Here is the list of faulty entries:\n%s"
+                "\nThese entries can easily be found checking the stated index in the produced ZonedInput.xlsx"
+                % (
+                    group["Zone"].iloc[0],
+                    element.name,
+                    group["OuterWallConstruction"].iloc[0],
+                    group,
+                )
+            )
+
+    bldg = Building(parent=project)
+    bldg.name = building_name
+    bldg.year_of_construction = construction_age
+    bldg.with_ahu = True  # HardCodedInput
+    if bldg.with_ahu is True:
+        bldg.central_ahu.heat_recovery = True  # HardCodedInput
+        bldg.central_ahu.efficiency_recovery = 0.35  # HardCodedInput
+        bldg.central_ahu.profile_temperature = 25 * [273.15 + 18]  # HardCodedInput
+        bldg.central_ahu.profile_min_relative_humidity = 25 * [0]  # HardCodedInput
+        bldg.central_ahu.profile_max_relative_humidity = 25 * [1]  # HardCodedInput
+        bldg.central_ahu.profile_v_flow = 25 * [1]  # HardCodedInput
+
+    # Parameters that need hard coding in teasers logic classes
+    # 1. "use_set_back" needs hard coding at aixlib.py in the init; defines
+    # if the in the useconditions stated
+    #   heating_time with the respective set_back_temp should be applied.
+    #   use_set_back = false -> all hours of the day
+    #   have same set_temp_heat actual value: use_set_back = Check!
+    # 2. HeaterOn, CoolerOn, hHeat, lCool, etc. can be hard coded in the text
+    # file
+    #   "teaser / data / output / modelicatemplate / AixLib /
+    #   AixLib_ThermalZoneRecord_TwoElement"
+    #   actual changes: Check!
+
+    # Parameters to be set for each and every zone (#HardCodedInput)
+    # -----------------------------
+    OutWallTilt = 90
```

Comment:
throughout the document could you please check for naming convention, e.g. as these are (hardcoded) but non global parameters it would be out_wall_tilt

Reply:
Thanks for noticing, variable naming is now according to PEP8.